source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
There are a few options then.

1) Abandon trying to do this... render only client side, move on
2) Put the include of this piece of crap into client, but DI it into your App perhaps on a prop?

```
// client.js
import mixPanel from 'mixpanel'

import App from './App'

render(<App  analyticsClient={mixPanel} />)

// server.js

import App from './App'

renderDocument({
  content: renderToStaticMarkup(<App />)
})

// App.js

const ({
  analtyicsClient = defaultImplemenation // so things will still work
})
```